### DISCUSSION TITLE
AdonisJs to use alternatives of Moment.js ?

### DISCUSSION THREAD

**Discussion by begueradj** [adonisjs/core#1598] [General]
The maintainers of Moment.js which Adonis Js uses internally [advice to use other alternatives](https://momentjs.com/docs/#/-project-status/recommendations/).

How this would impact AdonisJs ?

Thank you,
Billal BEGUERADJ

**Comment by RomainLanz** [comment]
Hey @begueradj! 👋 

We don't use `moment.js` anymore in AdonisJS 5 (which is available since March).
